.github/workflows/nightly_tests.yaml

```diff
@@ -0,0 +1,81 @@
+name: "CLC nightly tests"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * *'
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
+    runs-on: ${{ matrix.os }}
+    env:
+      GOPATH: "${{ github.workspace }}"
+      HZ_VERSION: "5.3.0-BETA-2"
+      DEFAULT_TIMEOUT: "30s"
+      ENABLE_VIRIDIAN: "1"
+      HZ_CLOUD_COORDINATOR_BASE_URL: "https://api.dev2.viridian.hazelcast.cloud"
+      # Note:
+      # VIRIDIAN_API_KEY and VIRIDIAN_API_SECRET are repo secrets
```

Comment:
How is the test going to access the secrets if they are not made environment variables? I believe we need to assign secrets to env variables.

Reply:
You are right, added them at: https://github.com/hazelcast/hazelcast-commandline-client/pull/226/commits/9eb193af8b9bf70c8552eba888d78a395697113d